docs/source/user/parallel.rst

```diff
@@ -82,7 +82,12 @@ supported reductions.
 A reduction is inferred automatically if a variable is updated by a binary
 function/operator using its previous value in the loop body. The initial value
 of the reduction is inferred automatically for the ``+=``, ``-=``,  ``*=``,
-and ``/=`` operators.
```

Comment:
Should this section explicitly declare which reductions are supported, I think the opening statement is a bit too general given what is available in practice?

Reply:
Done.